node/pkg/db/rediscriber.go

```diff
@@ -0,0 +1,112 @@
+package db
+
+import (
+	"context"
+
+	errorsentinel "bisonai.com/orakl/node/pkg/error"
+	"github.com/redis/go-redis/v9"
+	"github.com/rs/zerolog/log"
+)
+
+type Rediscriber struct {
+	client        *redis.Client
+	router        func(*redis.Message) error
+	redisChannels []string
```

Comment:
how about renaming `redisChannels` to `redisTopics` to avoid confusion since channel is also a builtin `go` concept?